FWCore/Framework/interface/ModuleProcessName.h

```diff
@@ -0,0 +1,30 @@
+#ifndef FWCore_Framework_ModuleProcessName_h
+#define FWCore_Framework_ModuleProcessName_h
+
+#include <string_view>
+
+namespace edm {
+  /**
+   * Helper class to hold a module label and a process name
+   *
+   * Note: does NOT own the string storage, be careful to use.
+   */
+  class ModuleProcessName {
```

Comment:
As ModuleDescription already holds both these pieces of information, why have this data structure as well?

Reply:
Good question. In this class the process name may be empty. On the other hand I could  move the process matching logic now in
https://github.com/cms-sw/cmssw/blob/feca570fcc574c514926e7ef758a4a0b52477334/FWCore/Framework/src/PathsAndConsumesOfModules.cc#L198-L201
to be done in `EDConsumerBase::modulesWhoseProductsAreConsumed()` and return `ModuleDescription` instead. But to use `ModuleDescription` I would need to get a hold of one of a parent (Sub)Process in an `EDConsumerBase::modulesWhoseProductsAreConsumed()` of a SubProcess. I'm not sure what would be the best way to do that.

Reply:
We agreed to keep `ModuleProcessName` for now.